receiver/jmxreceiver/config.go

```diff
@@ -278,3 +354,123 @@ func listKeys(presenceMap map[string]struct{}) string {
 	sort.Strings(list)
 	return strings.Join(list, ", ")
 }
+
+func (c *Config) buildJMXConfig() (string, error) {
```

Comment:
For reviewers: this function was moved from the `receiver.go` file (just deals with the Config struct).